src/components/DragAndDrop/index.js

```diff
@@ -21,7 +21,16 @@ const propTypes = {
 
 const defaultProps = {
     onDragOver: () => {},
-    shouldAcceptDrop: () => true,
+    shouldAcceptDrop: (e) => {
+        if (e.dataTransfer.types) {
+            for (let i = 0; i < e.dataTransfer.types.length; i++) {
+                if (e.dataTransfer.types[i] === 'Files') {
+                    return true;
+                }
+            }
+        }
+        return false;
+    },
```

Comment:
Let's just put it back where it was before. It is not drying up the code. That was more readable. It is fine to have duplicate code in the story.

Reply:
Shouldn't DragAndDrop work with files by default? I think that is the main use case

Reply:
Yeah, I agree but it seems like a lot of code for a default prop. you can also create a function and use it here.

Reply:
That's why i said earlier that it does not make sense to create empty functions as default props at all but ok.

Is it final decision to revert back the change and have clients specify this file validation function?

Reply:
> Is it final decision to revert back the change and have clients specify this file validation function?

Let's keep it this way for now.

Reply:
Oops, just checking now. My previous comment is wrong :facepalm: . I meant to say keep the old logic 1.e. revert back to the change.